src/liapunovspectrum.jl

```diff
@@ -80,12 +70,11 @@ function classicalGS(A)
     return Q, R
 end
 # Modified Gram–Schmidt (Trefethen algorithm 8.1)
-function modifiedGS(A)
+function modifiedGS!(A, Q, R, aⱼ, qᵢ)
     m,n = size(A)
-    Q = similar(A)
-    R = zeros(eltype(A),n,n)
-    aⱼ = Array{eltype(A)}(m)
-    qᵢ = similar(aⱼ)
+    R[:,:] = zeros(eltype(A),n,n)
```

Comment:
Just tried various combinations of `R`, `R[:,:]`, `zeros(eltype(A),n,n)` and `view(zeros(eltype(A),n,n))`, but the only combination that seems to work is the one already implemented